aesara/scan/basic.py

```diff
@@ -751,7 +751,7 @@ def wrap_into_list(x):
             # defined in scan utils
             sit_sot_scan_inputs.append(
                 expand_empty(
-                    at.unbroadcast(shape_padleft(actual_arg), 0),
+                    mask_broadcastable(shape_padleft(actual_arg), 0),
```

Comment:
The issue here seems to be creating an EmptyAlloc that does not have a fixed shape of 1 on the leftmost dimension. I didn't dig to find out why/whether this is actually needed